### PR TITLE
[Mellanox] Add device files for SN5600

### PR DESCRIPTION
**Pull request by Junchao-Mellanox**
#### Why I did it

Add device files for new platform SN5600

#### How I did it

Add device files for new platform SN5600

#### How to verify it

Manual test

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205

### CODE REVIEW

**Comment by keboliu** [comment]
Is the copyright header newly generated? it should be 2022 and there is a tool to generate it.

File: device/mellanox/x86_64-nvidia_sn5600-r0/system_health_monitoring_config.json [hunk 1]
```diff
@@ -0,0 +1 @@
+../x86_64-mlnx_msn2700-r0/system_health_monitoring_config.json
```
Comment (keboliu) [inline device/mellanox/x86_64-nvidia_sn5600-r0/system_health_monitoring_config.json:1]: file missing?

File: device/mellanox/x86_64-nvidia_sn5600-r0/thermal_policy.json [hunk 2]
```diff
@@ -0,0 +1 @@
+../x86_64-mlnx_msn2700-r0/thermal_policy.json
```
Comment (keboliu) [inline device/mellanox/x86_64-nvidia_sn5600-r0/thermal_policy.json:1]: wrong file path?

File: device/mellanox/x86_64-nvidia_sn5600-r0/pmon_daemon_control.json [hunk 3]
```diff
@@ -0,0 +1 @@
+../x86_64-mlnx_msn2700-r0/pmon_daemon_control.json
```
[the remark this answers is not in the document]
Comment (Junchao-Mellanox) [reply device/mellanox/x86_64-nvidia_sn5600-r0/pmon_daemon_control.json:1]: this is a soft link to default 2700 file.

File: device/mellanox/x86_64-nvidia_sn5600-r0/thermal_policy.json (hunk 2 above)
Comment (Junchao-Mellanox) [reply device/mellanox/x86_64-nvidia_sn5600-r0/thermal_policy.json:1]: this is a soft link to default 2700 file.

File: device/mellanox/x86_64-nvidia_sn5600-r0/system_health_monitoring_config.json (hunk 1 above)
Comment (Junchao-Mellanox) [reply device/mellanox/x86_64-nvidia_sn5600-r0/system_health_monitoring_config.json:1]: this is a soft link to default 2700 file.

**Review by keboliu — APPROVED** [review]

**Pull request closed without merging**